packages/simplescreenrecorder/PKGBUILD

```diff
@@ -0,0 +1,42 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: this.ven <misgi@ven.uber.space>
+
+_name=ssr
+pkgname=simplescreenrecorder
+pkgver=0.4.4
+pkgrel=1
+pkgdesc='A feature-rich screen recorder that supports X11 and OpenGL'
+arch=(x86_64 aarch64)
+url='https://www.maartenbaert.be/simplescreenrecorder'
+license=(GPL3)
+groups=(pro-audio)
```

Comment:
I wouldn't classify ssr as audio software, so I don't think the group fits here and should be removed.